src/components/modulesList/Item.js

```diff
@@ -1,37 +1,49 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
+import { Icon } from "react-native-elements";
 import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
-import { AntDesign } from "@expo/vector-icons";
 import { Colors, Typography } from "../../constants/styles";
+import { DiagnosisContext } from "../../modules/context/DiagnosisContext";
+
+const ModuleItem = ({ moduleCode, onPress }) => {
+  const { modules } = useContext(DiagnosisContext);
+  const module = modules[moduleCode];
+  const iconColor = module.wasVisited
+    ? Colors.GRAY_LIGHT
+    : Colors.GRAY_VERY_LIGHT;
 
-const ModuleItem = ({ module, onPress, conditionMet }) => {
-  const iconName = conditionMet === 1 ? "check" : "close";
-  const iconColor = conditionMet === 1 ? Colors.GREEN : Colors.RED;
   return (
-    <View style={styles.container}>
+    <View style={[styles.container, { backgroundColor: iconColor }]}>
       <TouchableOpacity style={styles.button} onPress={onPress}>
         <Text style={styles.text}>{module.name}</Text>
       </TouchableOpacity>
-      {conditionMet != null && (
-        <AntDesign
-          style={styles.icon}
-          name={iconName}
-          size={24}
-          color={iconColor}
-        />
-      )}
+      {module.diagnosis &&
+        module.diagnosis.map((result) => {
+          return (
+            <View style={styles.result} key={result.disease_icd10}>
+              <Text>
```

Comment:
kolor tekstu powinien być fioletowy

Reply:
Ogólnie to jest do wystylizowania. Kolor tła też bym zmienił i napisy się źle skalują. Zrobię taska na to i zajmę się tym na końcu. Jak będziesz mieć jakieś inne uwagi to chętnie przyjmę.